warehouse/query-core/src/test/java/datawave/query/tables/facets/FacetedQueryLogicTest.java

```diff
@@ -164,11 +164,9 @@ public void testQueryDynamicFacets() throws Exception {
         // TODO: this test isn't working properly. I would expect a query for Italy that is configured to facet
         // the CITY field - to return a facet for rome and paris, but also return a field name.
         Set<String> expected = new TreeSet<>();
-
-        // @formatter:off
         expected.add("null; paris -- paris//1");
         expected.add("null; rome -- rome//2");
-        // @formatter:on
+        expected.add("null; italy -- italy//10");
```

Comment:
@drewfarris any thoughts on why this would come back? I was a little concerned that what is effectively a no-op refactor caused more data to come back on this test, but then I saw the comment that it wasn't working properly. Is this enough to stop a merge?

Reply:
Nevermind, I found the answer and added a comment to the code.